scripts/setup/autoenv.sh

```diff
@@ -29,12 +29,12 @@ fi
 
 
 if [ -z "${HOLBA_DIR}" ]; then
-  HOLBA_DIR=${SETUP_DIR}/../..
+  HOLBA_DIR="${SETUP_DIR}/../.."
```

Comment:
Do you think the quotes are necessary here? I consistently didn't put any in these places :)

Reply:
Ah? Oh. Oops? I don't think that they are necessary.

**However**, I just noticed that this `if` is using single `[ ]`.

Reply:
Oh, true, I didn't really look for this anyways, you could grep something like "^[[ " and " ]^]" where ^ should be the not symbol. This to find the cases you missed.

But I meant it like a question, do you think there is an advantage of using quotes or a disadvantage of not using them for paths? I used them everywhere else consistently (I tried to be consistent).

Reply:
I'm not sure, but I would say that it's safer to use quotes everywhere, because there could be a space in `$SETUP_DIR`. Here it shouldn't pose a problem, but you really want to quote in `rm -rf "$SETUP_DIR/../../some/path/"`. Then, if you want to be consistent, you should quote everywhere :slightly_smiling_face: